README.rst

```diff
@@ -137,8 +137,36 @@ with ``td_spark_path`` option would be helpful.
    writer = SparkWriter(apikey='1/XXX', endpoint='https://api.treasuredata.com/', td_spark_path='/path/to/td-spark-assembly.jar')
    client.load_table_from_dataframe(df, 'mydb.bar', writer=writer, if_exists='overwrite')
 
+Comparison between pytd, td-client-python, and pandas-td
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Treasure Data offers three different Python clients on GitHub, and the following list summarizes their characteristics.
+
+1. `td-client-python <https://github.com/treasure-data/td-client-python>`__
+
+   - Basic REST API wrapper.
+   - Similar functionalities to td-client-{`ruby <https://github.com/treasure-data/td-client-ruby>`__, `java <https://github.com/treasure-data/td-client-java>`__, `node <https://github.com/treasure-data/td-client-node>`__, `go <https://github.com/treasure-data/td-client-go>`__}.
+   - The capability is limited by `what Treasure Data REST API can do <https://tddocs.atlassian.net/wiki/spaces/PD/pages/1085354/REST+APIs+in+Treasure+Data>`__.
+
+2. `pandas-td <https://github.com/treasure-data/pandas-td>`__ *(deprecated)*
```

Comment:
Since we've already deprecated pandas-td, it'd be better for pandas-td to move 3rd position to reduce customers' attention.

Reply:
Reordered 5663a0175c0508ca2e532320f07be5c1833ccf5a